docs/detections/prebuilt-rules/downloadable-packages/0-16-1/prebuilt-rule-0-16-1-suspicious-portable-executable-encoded-in-powershell-script.asciidoc

```diff
@@ -0,0 +1,125 @@
+[[prebuilt-rule-0-16-1-suspicious-portable-executable-encoded-in-powershell-script]]
+=== Suspicious Portable Executable Encoded in Powershell Script
+
+Detects the presence of portable executables (PE) in a PowerShell script by looking for its encoded header. Attackers embed PEs into PowerShell scripts for injecting them into the memory, avoiding defenses by not writing to disk.
+
+*Rule type*: query
+
+*Rule indices*: 
+
+* winlogbeat-*
+* logs-windows.*
+
+*Severity*: medium
+
+*Risk score*: 47
+
+*Runs every*: 5m
+
+*Searches indices from*: now-9m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)
+
+*Maximum alerts per execution*: 100
+
+*References*: 
+
+* https://github.com/atc-project/atc-data/blob/master/docs/Logging_Policies/LP_0109_windows_powershell_script_block_log.md
+
+*Tags*: 
+
+* Elastic
+* Host
+* Windows
+* Threat Detection
+* Execution
+
+*Version*: 4
+
+*Rule authors*: 
+
+* Elastic
+
+*Rule license*: Elastic License v2
+
+
+==== Investigation guide
+
+
+[source, markdown]
+----------------------------------
+## Triage and analysis.
+
+### Investigating Suspicious Portable Executable Encoded in Powershell Script
+
+PowerShell is one of the main tools used by system administrators for automation, report routines, and other tasks.
+
+Attackers can abuse PowerShell in-memory capabilities to inject executables into memory without touching the disk,
+bypassing antivirus software. These executables are generally base64 encoded.
+
+#### Possible investigation steps:
+
+- Examine script content that triggered the detection. 
+- Investigate script execution chain (parent process tree).
+- Inspect any file or network events from the suspicious PowerShell host process instance.
+- If the action is suspicious for the user, check for any other activities done by the user in the last 48 hours.
+
+### False Positive Analysis
+
+- Verify whether the script content is malicious/harmful.
+
+### Related Rules
+
+- PowerShell Reflection Assembly Load - e26f042e-c590-4e82-8e05-41e81bd822ad
+- PowerShell Suspicious Payload Encoded and Compressed - 81fe9dc6-a2d7-4192-a2d8-eed98afc766a
+- PowerShell PSReflect Script - 56f2e9b5-4803-4e44-a0a4-a52dc79d57fe
+
+### Response and Remediation
+
+- Immediate response should be taken to validate, investigate, and potentially contain the activity to prevent further
```

Comment:
need a find and replace for "Immediate response should be taken"->"Immediate steps should be taken"